strax/processing/general.py

```diff
@@ -389,7 +389,7 @@ def _check_things_containers(things, containers, length_threshold=0, sorted_endt
     if sorted_endtime:
         _check_sorted_endtime(things, containers)
     _check_length(things, containers, length_threshold)
-    _check_overlapping(things, containers)
+    # _check_overlapping(things, containers)
```

Comment:
We should still check that the containers are not overlapping in `fully_contained_in`.